content/source/docs/enterprise/api/workspaces.html.md

```diff
@@ -116,35 +98,40 @@ $ curl \
 }
 ```
 
-## Create a Workspace Without a VCS Repository
-
-The default `/workspaces` endpoint creates a workspace without configuring the VCS connection (`ingress-trigger`).
-
-| Method | Path           |
-| :----- | :------------- |
-| POST | /organizations/:organization_username/workspaces |
+### Create a Workspace with a VCS Repository
 
+By supplying the necessary attributes under a `vcs-repository` object, you can create a Workspace that is configured against a VCS Repository.
 
+#### Parameters
 
-### Parameters
-
-- `:organization_username` (`string: <required>`) - Specifies the username or organization name under which to create the workspace. The organization must already exist in the system, and the user must have permissions to create new workspaces. This parameter is specified in the URL path.
+- `name` (`string: <required>`) - Specifies the name of the workspace, which can only include letters, numbers, `-`, and `_`. This will be used as an identifier and must be unique in the organization.
+- `working-directory` (`string: ''`) - Specifies the directory that Terraform will execute within. This defaults to the root of your repository and is typically set to a subdirectory matching the environment when multiple environments exist within the same repository.
```

Comment:
I realize this is carried over verbatim, but 1. could use some clarification about what the empty string means, and 2. could stand to specify that this is a relative path.

Reply:
I'm guessing that these all take the form of `type: default_value|<optional>` so my guess is that it's just that we default it to an empty string which doesn't do anything. I'll update the relative path bit though!